chore: update issue verifier (#43339)
vercel/next.js: .github/actions/issue-validator/canary.md

Please verify that your issue can be recreated with `next@canary`.

### **Why was this issue marked with the `please verify canary` label?**

> We noticed the provided reproduction was using an older version of Next.js, instead of `canary`.

Reviewer: ```suggestion
We noticed the provided reproduction was using an older version of Next.js, instead of the latest `canary`.
```

### **How can I quickly verify if my issue has been fixed in `canary`?**

The safest way is to install `next@canary` in your project and test it, but you can also search through [closed Next.js issues](https://github.com/vercel/next.js/issues?q=is%3Aissue+is%3Aclosed) for duplicates or check the [Next.js releases](https://github.com/vercel/next.js/releases).

### **My issue has been open for a long time, why do I need to verify `canary` now?**

Next.js does not backport bug fixes to older versions of Next.js. Instead, we are trying to introduce only a minimal amount of breaking changes between major releases.

### **What happens if I don't verify against the canary version of Next.js?**

An issue with the `please verify canary` that receives no meaningful activity (e.g. new comments that acknowledge verification against `canary`) will be automatically closed and locked after 30 days.

If your issue has not been resolved in that time and it has been closed/locked, please open a new issue, with the required reproduction, using `next@canary`.

### **I did not open this issue, but it is relevant to me, what can I do to help?**

> Anyone experiencing the same issue is welcome to provide a minimal reproduction (following the above steps). Furthermore, you can upvote the issue (using the :+1: reaction on the topmost comment, instead of commenting "I have the same issue" etc.). Then, we can sort issues by engagement to prioritize.

Reviewer: ```suggestion
Anyone experiencing the same issue is welcome to provide a minimal reproduction following the above steps. Furthermore, you can upvote the issue using the :+1: reaction on the topmost comment (please **do not** comment "I have the same issue" without repro steps). Then, we can sort issues by votes to prioritize.
```
